components/esb-tools/plugins/org.wso2.integrationstudio.esb.form.editors/src/org/wso2/integrationstudio/esb/form/editors/article/providers/ValueExpressionWidgetNameSpacedPropertyEditorDialog.java

```diff
@@ -0,0 +1,1746 @@
+            
+            // Configure group box internal layout.
+            FormLayout jsonEvaluatorGroupBoxLayout = new FormLayout();
+            jsonEvaluatorGroupBoxLayout.marginWidth = 5;
+            jsonEvaluatorGroupBoxLayout.marginHeight = 5;
+            jsonEvaluatorGroup.setLayout(jsonEvaluatorGroupBoxLayout);
+            
+            jsonEvaluatorGroup.setText("Evaluate JSON Path");
+            jsonEvaluatorGroup.setVisible(false);
+        }
+        
+        exprEvaluatorGroup = new Group(dialogShell, SWT.NONE);
+        {   
+            FormData exprEvaluatorGroupBoxLayoutData = new FormData();
+            exprEvaluatorGroupBoxLayoutData.top = new FormAttachment(expressionGroup, 2);
+            exprEvaluatorGroupBoxLayoutData.left = new FormAttachment(1);
+            exprEvaluatorGroupBoxLayoutData.right = new FormAttachment(99);
+            exprEvaluatorGroupBoxLayoutData.bottom = new FormAttachment(okButton, -5);
+            exprEvaluatorGroup.setLayoutData(exprEvaluatorGroupBoxLayoutData);
+            
+            // Configure group box internal layout.
+            FormLayout exprEvaluatorGroupBoxLayout = new FormLayout();
+            exprEvaluatorGroupBoxLayout.marginWidth = 5;
+            exprEvaluatorGroupBoxLayout.marginHeight = 5;
+            exprEvaluatorGroup.setLayout(exprEvaluatorGroupBoxLayout);
+            
+            exprEvaluatorGroup.setVisible(false);
+        }
+        
+        exprBrowserView = new Browser(exprEvaluatorGroup, SWT.NONE);
+        {
+            FormData exprBrowserViewLayoutData = new FormData();
+            exprBrowserViewLayoutData.top = new FormAttachment(2);
+            exprBrowserViewLayoutData.left = new FormAttachment(1);
+            exprBrowserViewLayoutData.right = new FormAttachment(99);
+            exprBrowserViewLayoutData.bottom = new FormAttachment(99);
+            exprBrowserView.setLayoutData(exprBrowserViewLayoutData);
+            
+            // Configure group box internal layout.
+            FormLayout exprBrowserViewLayout = new FormLayout();
+            exprBrowserViewLayout.marginWidth = 5;
+            exprBrowserViewLayout.marginHeight = 5;
+            exprBrowserView.setLayout(exprBrowserViewLayout);
+            
+            URL url;
+            try {
+                url = new URL(EXPR_HELP_HTML_FILE_PATH);
+                
+                InputStream inputStream = url.openConnection().getInputStream();
+                BufferedReader in = new BufferedReader(new InputStreamReader(inputStream));
+                String inputLine = "";
+                String html = "";
+                
+                while ((inputLine = in.readLine()) != null) {
+                    html += inputLine;
+                }
+                
+                exprBrowserView.setBounds(100, 100, 1000, 800);
+                exprBrowserView.setText(html);
+                in.close();
```

Comment:
What will happen when an exception is thrown? Shall we move this to a finally block?

Reply:
Fixed in https://github.com/wso2/integration-studio/pull/1114